ocs_ci/helpers/helpers.py

```diff
@@ -750,6 +750,7 @@ def default_storage_class(
         )
         cmd = f"oc patch storageclient ocs-storagecluster --subresource status --type merge -p '{patch}'"
         exec_cmd(cmd)
+        time.sleep(60)
```

Comment:
Should we log the reason for waiting a minute?

Reply:
done